<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>49</y>
     <width>1051</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Diffraction Condition</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="6">
         <widget class="QLineEdit" name="lineEdit_lambda">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Lambda</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="lineEdit_2thetaA">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_sample"/>
        </item>
        <item row="1" column="7">
         <widget class="QLineEdit" name="lineEdit_180_2thetaa">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sample</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEdit_analyzer_d">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>2θ_S</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2thetaS"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBox_analyzer"/>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>180-2θ_A</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_sample_hkl_d">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>(hkl)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QLineEdit" name="lineEdit_GonA">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Si Analyzer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>2θ_A</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBox_sample_hkl"/>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Gon_A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Focusing Condition</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Vertical Curvature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>LSA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>LAD</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Horizontal Curvature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>Magnification</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>1/RH</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Analyzer Turns</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_RV"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_LSA"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_LAD">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_RH">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_mag"/>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEdit_RH_inv">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLineEdit" name="lineEdit_analyzer_turns">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
